tests/format/js/method-chain/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -1,5 +1,87 @@
+=====================================input======================================
+this.is.a.long.long.long.long.long.long.long.long.long.long.long.long.long.long.long.long.method();
+
+prettier.is.an.opinionated.code.formatter.with.support.for(
+  "JavaScript (including experimental features)",
+  "JSX",
+  "Angular",
+  "Vue",
+  "Flow",
+  "TypeScript",
+  "CSS, Less and SCSS",
+  "HTML",
+  "Ember/Handlebars",
+  "JSON",
+  "GraphQL",
+  "Markdown, including GFM and MDX",
+  "YAML")
+  .it.removes.all.original.styling.and.ensures.that.all.outputted
+  .code.conforms.to.a.consistent.style(see, this, blogpost);
+
+By.far.the.biggest.reason.for.adopting.Prettier.is.to.stop.all.the['on-going'].debates.over.styles()
+.It.is.generally.accepted.that.having.a.common.style.guide.is.valuable.for.a.project.and.team.but.getting.there.is.a.very.painful.and.unrewarding.process()
+.People.get.very.emotional.around.particular.ways.of.writing.code.and.nobody.likes.spending.time.writing.and.receiving.nits()
+
+So.why.choose.the.Prettier./* comment */style.guide/* comment */.over.any.other.random.style.guide?.()
+.Because.Prettier.is./* comment */the.only./* comment */style.guide.that.is.fully.automatic()
+// commment
+.Even // comment
+.if.Prettier.does.not.format.all.code['100%'].the.way['you’d'].like['it’s'].worth.the.sacrifice.given.the.unique.benefits.of.Prettier
+['don’t'].you.think?.()
+
+=====================================output=====================================
+this.is.a.long.long.long.long.long.long.long.long.long.long.long.long.long.long
+  .long.long.method();
+
+prettier.is.an.opinionated.code.formatter.with.support
+  .for(
+    "JavaScript (including experimental features)",
+    "JSX",
+    "Angular",
+    "Vue",
+    "Flow",
+    "TypeScript",
+    "CSS, Less and SCSS",
+    "HTML",
+    "Ember/Handlebars",
+    "JSON",
+    "GraphQL",
+    "Markdown, including GFM and MDX",
+    "YAML",
+  )
+  .it.removes.all.original.styling.and.ensures.that.all.outputted.code.conforms
+    .to.a.consistent.style(see, this, blogpost);
+
+By.far.the.biggest.reason.for.adopting.Prettier.is.to.stop.all.the["on-going"]
+  .debates.over
+  .styles()
+  .It.is.generally.accepted.that.having.a.common.style.guide.is.valuable.for.a
+    .project.and.team.but.getting.there.is.a.very.painful.and.unrewarding
+    .process()
+  .People.get.very.emotional.around.particular.ways.of.writing.code.and.nobody
+    .likes.spending.time.writing.and.receiving.nits();
+
+So.why.choose.the.Prettier./* comment */ style.guide /* comment */.over.any
+  .other.random.style
+  .guide?.()
+  .Because.Prettier.is./* comment */ the.only./* comment */ style.guide.that.is
+    .fully.automatic()
+  // commment
+  .Even // comment
+  .if.Prettier.does.not.format.all.code["100%"].the.way["you’d"].like["it’s"]
+    .worth.the.sacrifice.given.the.unique.benefits.of.Prettier["don’t"].you
+    .think?.();
```

Comment:
Can you find some real world case? These tests doesn't make sense to me at all. Nobody write code like this.

Reply:
I see. I'll consider a more practical test case. This is just a test case that is so long chain and not likely to have a copyright / license issue.